package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/lab": "^4.0.0-alpha.56",
     "bootstrap": "^4.3.1",
```

Comment:
`bootstrap` should be remove since you're using `material-ui` and this can affect the application loading time.